projects/plugins/jetpack/extensions/blocks/sharing-buttons/editor.js

```diff
@@ -2,8 +2,31 @@ import { registerJetpackBlockFromMetadata } from '../../shared/register-jetpack-
 import metadata from './block.json';
 import edit from './edit';
 import save from './save';
+import './style.scss';
 
 registerJetpackBlockFromMetadata( metadata, {
 	edit,
 	save,
+	example: {
```

Comment:
I believe the example could be added to `block.json` instead, since it doesn't have dynamic values.